spec/rivets/binding.js

```diff
@@ -17,6 +17,27 @@ describe('Rivets.Binding', function() {
     binding.binder.routine.should.equal(rivets.binders.text)
   })
 
+  describe('when bind to non configurable properties', function() {
+    beforeEach(function () {
+      data = {
+        name: 'x',
+        items: []
+      }
+      Object.defineProperty(data, 'name', {
+        enumerable: true,
+        configurable: false,
+        writable: true,
+        value: 'y'
+      });
+      el.setAttribute('data-show', 'obj.items.length')
+    })
+
+    it("does not throw", function() {
+      view = rivets.bind(el, {obj: data});
```

Comment:
It does not have expect. Get:  ReferenceError: Can't find variable: expect
Managed in a different way